go.mod

```diff
@@ -28,11 +28,12 @@ require (
 	github.com/cosmos/ibc-apps/modules/rate-limiting/v8 v8.0.0
 	github.com/cosmos/ibc-go/modules/capability v1.0.1
 	github.com/cosmos/ibc-go/v8 v8.5.1
+	github.com/cosmos/interchain-security/v5 v5.0.0-20240904071545-7301916eeafa
```

Comment:
Changes made so that I can use `providerKeeper`s new methods in `upgrades.go`.
Should be updated later on.